game/src/game/components/audio/Listener.h

```diff
@@ -0,0 +1,23 @@
+#pragma once
+
+#include "engine/audio/AudioService.h"
+#include "engine/scene/Component.h"
+#include "engine/scene/OnUpdateEvent.h"
+#include "engine/scene/Transform.h"
+
+class Listener final : public Component, public IEventSubscriber<OnUpdateEvent>
```

Comment:
Could we rename this to `AudioListener` instead? if you're just looking at the class name when it's added as a component it's a bit hard to tell if it's related to events or not. imo `Listener` could be mistaken as being the same as `EventSubscriber`,  i.e. `EventListener`)

Reply:
yeah, i was thinking this too